lib/logmsg/logmsg.c

```diff
@@ -524,9 +524,9 @@ log_msg_set_value(LogMessage *self, NVHandle handle, const gchar *value, gssize
   if (_log_name_value_updates(self))
     {
       msg_debug("Setting value",
-                evt_tag_printf("msg", "%p", self),
-                evt_tag_str("name", name),
-                evt_tag_printf("value", "%.*s", (gint) value_len, value));
+                evt_tag_str("macro", name),
+                evt_tag_printf("value", "%.*s", (gint) value_len, value),
```

Comment:
This feels like it could be a new evtlog function, say `evt_tag_strn(const char *tag, gint value_len, const char *value)`... It may not be used a lot, but having a dedicated function for this feels much cleaner.

Mind you, this may not need to be part of *this* pull request, but... let this be a food for thought!

Reply:
Agreed.